lib/chef/resource.rb

```diff
@@ -811,7 +812,10 @@ def updated=(true_or_false)
     #
     # @return [String] The DSL name of this resource.
     def self.dsl_name
-      convert_to_snake_case(name, 'Chef::Resource')
```

Comment:
I would like to normalize *_name, but it didn't seem like the time :)  resource_name doesn't exist on Provider, at any rate--it's only on the class in LWRPBase.